CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ All notable changes to this project will be documented in this file.
 
 - Introduced a new filtering system using classes for more flexible and robust filtering options
 - Added 'get_jobs' method, a feature to retrieve a list of jobs based on 'JobFilters'
+- Added 'get_batches' method, a feature to retrieve a list of batches based on 'BatchFilters'
```

Comment:
So in this case you would need to add a new section with a Added subtitle and a newer version (add +1 to the patch version of the latest released version).